### PR TITLE
New google icons

### LINKED ISSUE #458: New Google Logos!
Google is getting some *horrible* new logos. https://techcrunch.com/2020/10/06/googles-new-logos-are-bad/

They are not yet available at https://www.google.com/permissions/trademark/logos-list/

Non-vector logos at https://workspace.google.co.uk/intl/en_uk/

Anyone have a source of the vectors?

### PR DESCRIPTION
**Pull request by jonatjano**
I added the 5 new google logos

closes #458

### CODE REVIEW

File: images/svg/gmail.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg"
-aria-label="Gmail" role="img"
+aria-label="google gmail" role="img"
 viewBox="0 0 512 512"><rect
 width="512" height="512"
 rx="15%"
-fill="#fff"/><path fill="#f2f2f2" d="M120 392V151.075h272V392"/><path fill-opacity=".05" d="M256 285L120 392l-4-212"/><path fill="#d54c3f" d="M120 392H97c-12 0-22-10-22-23V143h45z"/><path fill-opacity=".08" d="M317 392h77V159H82"/><path fill="#f2f2f2" d="M97 121h318L256 234"/><path fill="#b63524" d="M392 392h23c12 0 22-10 22-23V143h-45z"/><path fill="none" stroke="#de5145" stroke-linecap="round" stroke-width="44" d="M97 143l159 115 159-115"/></svg>
+fill="white"/><path d="M158 391v-140l-82-63V361q0 30 30 30" fill="#4285f4"/><path d="M76 188l82 63v-98l-30-23c-27-21-52 0-52 26" fill="#c5221f"/><path d="M158 251l98 74l98-74v-98l-98 74l-98-74" fill="#ea4335"/><path d="M436 188l-82 63v-98l30-23c27-21 52 0 52 26" fill="#fbbc04"/><path d="M354 391v-140l82-63V361q0 30-30 30" fill="#34a853"/></svg>
```
Comment (edent) [inline images/svg/gmail.svg]: `#fff` is shorter than `white`

File: images/svg/gmail.svg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg"
-aria-label="Gmail" role="img"
+aria-label="google gmail" role="img"
```
Comment (edent) [inline images/svg/gmail.svg]: I think just `Gmail` is fine.

File: images/svg/google_drive.svg [hunk 3]
```diff
@@ -1,8 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg"
-aria-label="Google Drive" role="img"
+aria-label="google drive" role="img"
```
Comment (edent) [inline images/svg/google_drive.svg]: Keep the capitalisation, please.

File: images/svg/google_meet.svg [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="google meet" role="img"
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="white"/><path d="M76 106h360v300h-360z" fill="transparent" stroke="white"/><path d="M166 106v90h-90" fill="#ea4335"/><path d="M166 106v90h120v60l90-73v-47q0-30-30-30" fill="#ffba00"/><path d="M286 256l90-73v146" fill="#00832d"/>
+<path d="M376 183l42-34c9-7 18-7 18 7v200c0 14-9 14-18 7l-42-34" fill="#00ac47"/>
```
Comment (edent) [inline images/svg/google_meet.svg]: These newlines can be removed.

**Comment by jonatjano** [comment]
I did the requested change, it should be good now.

**Review by Forrest-J — COMMENTED** [review]
"Google collaborative content tools" should be "Google Docs Editors"

**Comment by jonatjano** [comment]
if you look on https://www.blog.google/press/ you'll see that Google Docs still has the blue icon :
image
image

**Comment by Forrest-J** [comment]
In some places & historically Google also calls the suite itself "[Google Docs Editors](https://support.google.com/docs/?hl=en#topic=1382883)" as opposed to "Google Docs", the word processor. See also [Wikipedia](https://en.wikipedia.org/wiki/Google_Docs_Editors).

**Comment by jonatjano** [comment]
I didn't know that, I wonder what would be the best way of solving this.

I can imagine 4 possible solutions : 
  1) keep `Collaborative content tools`, user may search `Docs editor`
  2) rename it `Docs Editors`, user may search `collaborative content tools`
  3) have both, need to modify both when editing
  4) make a symlink, one of the two will have the wrong `aria-label`

I think 3) has the lesser drawback since google shouldn't change the icon again for a while, nor change it multiple times in a short period of time

**Comment by edent** [comment]
I say option 3. I'm also thinking about keeping the older versions as an option. Just in case Google changes its mind.

**Comment by jonatjano** [comment]
We'll keep the older version anyways since it's not `Google Docs` which got the new icon but `Google Docs Editor`, this one refers to the whole suite (Docs, Calc, Slides, ...)

**Comment by Forrest-J** [comment]
Also from how I understand it "Collaborative content tools" isn't the proper name but just a descriptor given that it is set in sentence case, whereas "Google Docs Editors" is the proper name.

**Comment by jonatjano** [comment]
I found some things : 
1) from [google trademark list](https://www.google.com/permissions/trademark/trademark-list/) there is no `Google docs editors` nor `Google collaborative content tools` but there is `G suite` with is described as "productivity and collaboration tools"
2) looking for `G suite` I found https://workspace.google.fr/intl/fr/ which tell us `Gsuite` was renamed into `Google Workspace`
3) going back to the trademark list, there is no `Google Workspace` in there

also, the [trademark logo list](https://www.google.com/permissions/trademark/logos-list/) is not up to date

File: images/svg/gmail.svg [hunk 5]
```diff
@@ -3,4 +3,4 @@ aria-label="Gmail" role="img"
 viewBox="0 0 512 512"><rect
 width="512" height="512"
 rx="15%"
-fill="#fff"/><path fill="#f2f2f2" d="M120 392V151.075h272V392"/><path fill-opacity=".05" d="M256 285L120 392l-4-212"/><path fill="#d54c3f" d="M120 392H97c-12 0-22-10-22-23V143h45z"/><path fill-opacity=".08" d="M317 392h77V159H82"/><path fill="#f2f2f2" d="M97 121h318L256 234"/><path fill="#b63524" d="M392 392h23c12 0 22-10 22-23V143h-45z"/><path fill="none" stroke="#de5145" stroke-linecap="round" stroke-width="44" d="M97 143l159 115 159-115"/></svg>
+fill="#fff"/><path d="M158 391v-140l-82-63V361q0 30 30 30" fill="#4285f4"/><path d="M76 188l82 63v-98l-30-23c-27-21-52 0-52 26" fill="#c5221f"/><path d="M158 251l98 74l98-74v-98l-98 74l-98-74" fill="#ea4335"/><path d="M436 188l-82 63v-98l30-23c27-21 52 0 52 26" fill="#fbbc04"/><path d="M354 391v-140l82-63V361q0 30-30 30" fill="#34a853"/></svg>
```
Comment (logiclrd) [inline images/svg/gmail.svg]: There seems to be an imperfection between the dark and the light red, a subpixel band of white peeking through.

Comment (logiclrd) [reply images/svg/gmail.svg]: I can't find it when I zoom in, and all the shapes are specified in coordinates given in whole numbers, so it must be a rendering quirk, but:

image

File: images/svg/google_meet.svg [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="Google Meet" role="img"
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="#fff"/><path d="M76 106h360v300h-360z" fill="transparent" stroke="white"/><path d="M166 106v90h-90" fill="#ea4335"/><path d="M166 106v90h120v60l90-73v-47q0-30-30-30" fill="#ffba00"/><path d="M286 256l90-73v146" fill="#00832d"/><path d="M376 183l42-34c9-7 18-7 18 7v200c0 14-9 14-18 7l-42-34" fill="#00ac47"/><path d="M166 406v-90h120v-60l90 73v47q0 30-30 30" fill="#00ac47"/><path d="M76 316v60q0 30 30 30h60v-90" fill="#0066da"/><path d="M76 196h90v120h-90" fill="#2684fc"/></svg>
```
Comment (logiclrd) [inline images/svg/google_meet.svg]: I see it in this one too:

image

**Comment by jonatjano** [comment]
It shouldn't happens, definitely a rendering quirk, I'll try to see if there is a way to avoid this when I have a little time. Probably not this weekend, I got a rush for a school project.

(edent, since the issue's not ready to be merged, could I get an `hacktoberfest-accepted` label please)

**Pull request merged**